docs/3.0rc/get-started/index.mdx

```diff
@@ -31,7 +31,7 @@ if __name__ == "__main__":
   <Card title="Prefect Cloud" icon="cloud" href="/3.0rc/manage/cloud/index">
     Supercharge Prefect with enhanced governance, security, and performance capabilities.
   </Card>
-  <Card title="Upgrade to Prefect 3" icon="sparkles" href="/3.0rc/resources/upgrade-prefect-3">
+  <Card title="Upgrade to Prefect 3" icon="sparkles" href="/3.0rc/resources/upgrade-to-prefect-3">
```

Comment:
Good catch here. The linked page is changed to **/upgrading-to-prefect-3** in this PR.  So either the link here or the linked page needs an update.

The thinking on standardizing on the imperative vs present participle was to try to keep the page names consistent and action oriented - that was Stellar's rec.

Reply:
Yep, let's use upgrade-to-prefect for consistency. I can add as suggestion to the PR.

Reply:
Done.

Reply:
Thank you!